tests/author.tests.js

```diff
@@ -0,0 +1,138 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const { Author, Book } = require('../src/models/index');
+const app = require('../src/app');
+
+describe('/authors', () => {
+  before(async () => {
+    await Author.sequelize.sync({ force: true });
+  });
+
+  beforeEach(async () => {
+    await Author.destroy({ where: {} });
+  });
+
+  describe('with no records in the database', () => {
+    describe('POST /authors', () => {
+      it('creates a new author in the database', async () => {
+        const response = await request(app).post('/authors').send({
+          author: 'Stephen King',
+        });
+        const newAuthorRecord = await Author.findByPk(response.body.id, {
+          raw: true,
+        });
+
+        expect(response.status).to.equal(201);
+        expect(response.body.author).to.equal('Stephen King');
+        expect(newAuthorRecord.author).to.equal('Stephen King');
+      });
+    });
+    it('throws an error if author is null', async () => {
```

Comment:
There should always be a clear line between each of your tests so its obvious where each end as and next begins:
```
    });

    it('throws an error if author is null', async () => {
```